dotNet/uuid_example/README.md

```diff
@@ -0,0 +1,14 @@
+### Example .NET c# program inserting and reading a uuid value using the ADO.NET provider for Actian.
+#### Preparation
+Environment variables:
+- DOTNET_CONNECTION_STRING - Connection String used to connect to the database _(required)_
+e.g: DOTNET_CONNECTION_STRING=Server=myserver;Port=II7;Database=mydb;User ID=ingres;Password=ca-ingres
+
+### In Visual Studio
+
+To run the test application in Visual Studio 2022 start from a Command Prompt: 
+
+```
```

Comment:
Super pedantic, I'm not super keen on backtick/fenced for markdown. It is an extension https://www.markdownguide.org/extended-syntax/#fenced-code-blocks and not core, so I'd rather see this be indented (4 spaces, rather than tabs) to be consistent with other README usage in other Actian repos.